enterprise/internal/campaigns/store_changeset_events_test.go

```diff
@@ -44,7 +49,7 @@ func testStoreChangesetEvents(t *testing.T, ctx context.Context, s *Store, _ rep
 		}
 
 		// Verify that no duplicates are introduced and no error is returned.
-		for i := 0; i < 2; i++ {
+		for i := 0; i < len(events); i++ {
```

Comment:
I know this is only a test, but this is a weird looking loop. Why is it being done `len(events)` number of times?

Reply:
whoups good catch. It's supposed to test the upsert function does actually upsert. Not needed to run more often 🤦 
```suggestion
		for i := 0; i < 2; i++ {
```